tests/ieee802154_hal/main.c

```diff
@@ -260,7 +312,47 @@ static int send(uint8_t *dst, size_t dst_len,
         .iol_next = NULL,
     };
 
-    flags = IEEE802154_FCF_TYPE_DATA | IEEE802154_FCF_ACK_REQ;
+    flags = IEEE802154_FCF_TYPE_DATA ; //IEEE802154_FCF_ACK_REQ
```

Comment:
we can reduce code duplication if we create one "send" function that gets called with different arguments